internal/cli/tables-to-go-cli.go

```diff
@@ -274,3 +309,23 @@ func indexCaseInsensitive(s, substr string) int {
 	s, substr = strings.ToLower(s), strings.ToLower(substr)
 	return strings.Index(s, substr)
 }
+
+// ValidVariableName checks for the existence of any characters
+// outside of Unicode letters, numbers and underscore.
+func validVariableName(str string) bool {
```

Comment:
Altough it's a private function we should provide some unit tests for it.

Can we use `s` instead if `str` as parameter name?

Reply:
Tests added in e954a4472d and 809e9861ff8
Parameter name changed in ef3a47706c65